community/dnscrypt-proxy/dnscrypt-proxy.pre-install

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
-addgroup -S dnscrypt 2>/dev/null
-adduser -S -D -H -h /var/empty -s /sbin/nologin -G dnscrypt -g dnscrypt dnscrypt 2>/dev/null
+addgroup -S dnscrypt 2>/dev/null || true
+adduser -S -D -H -h /var/empty -s /sbin/nologin -G dnscrypt -g dnscrypt dnscrypt 2>/dev/null || true
+
+install -d -o dnscrypt -g dnscrypt -m 755  /var/cache/dnscrypt-proxy 2>/dev/null 
+install -d -o dnscrypt -g dnscrypt -m 755  /var/log/dnscrypt-proxy 2>/dev/null
```

Comment:
This install script should not go here, but inside the APKBUILD.
See, for example, community/pgbouncer
Thanks!

Reply:
Gotcha - I've tidied up some mkdir's at the same time.
thanks for the pointer